packages/manager/apps/pci-private-network/src/routes.tsx

```diff
@@ -16,60 +16,50 @@ export interface RouteHandle {
   tracking?: string;
 }
 
-export const ROUTE_PATHS = {
+const ROUTE_PATHS = {
   root: '/pci/projects/:projectId/private-networks',
   onboarding: 'onboarding',
-  globalRegions: '',
+  listing: '',
   localZone: 'localZone',
   delete: 'delete',
   new: 'new',
 };
 
 export default [
   {
-    path: '/',
-    ...lazyRouteConfig(() => import('@/pages/Layout')),
-  },
-  {
-    id: 'private-networks',
     path: ROUTE_PATHS.root,
     loader: async ({ params }) =>
       queryClient.fetchQuery(getProjectQuery(params.projectId)),
     ...lazyRouteConfig(() => import('@/pages/Layout')),
     children: [
       {
-        path: ROUTE_PATHS.globalRegions,
-        handle: {
-          tracking: 'globalRegions',
-        },
-        ...lazyRouteConfig(() => import('@/pages/list/List.page')),
+        path: ROUTE_PATHS.listing,
+        ...lazyRouteConfig(() =>
+          import('@/pages/listing/ListingLayout.component'),
+        ),
         children: [
           {
-            path: ROUTE_PATHS.delete,
+            path: '',
             ...lazyRouteConfig(() =>
-              import('@/pages/delete/DeleteNetwork.page'),
+              import('@/pages/listing/region/PrivateNetworkRegion.page'),
             ),
-            handle: {
-              tracking: 'delete',
-            },
           },
-        ],
-      },
-      {
-        path: ROUTE_PATHS.localZone,
-        handle: {
-          tracking: 'localZone',
-        },
-        ...lazyRouteConfig(() => import('@/pages/list/List.page')),
-        children: [
           {
-            path: ROUTE_PATHS.delete,
+            path: ROUTE_PATHS.localZone,
             ...lazyRouteConfig(() =>
-              import('@/pages/delete/DeleteNetwork.page'),
+              import('@/pages/listing/localZone/PrivateNetworkLZ.page'),
             ),
```

Comment:
Routing seems weird, I think we should use query params instead but I don't understand how LZ is different from classic regions